docs/src/faqs/migrate_from_fnapi.rst

```diff
@@ -233,85 +80,60 @@ The following is a collection of suggested changes to optimize Docker builds by
 
-* **DELETE** any lines creating ``/usr/src/config``, i.e. ``RUN mkdir -p /usr/src/config``.
-* **DELETE** the two lines ``ARG KLIO_CONFIG=klio-job.yaml`` and ``COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml``.
+* **DELETE** lines copying ``MANIFEST.in`` and ``setup.py`` as they are no longer used.
 
 
 .. collapsible:: Example of Suggested Changes
 
     .. code-block:: diff
 
-           FROM apache/beam_python3.6_sdk:2.23.0
+           FROM apache/beam_python3.6_sdk:2.24.0
 
            WORKDIR /usr/src/app
-        -  RUN mkdir -p /usr/src/config
 
            ENV GOOGLE_CLOUD_PROJECT my-project \
               PYTHONPATH /usr/src/app
 
-        -  RUN apt-get update && apt-get install -y libsndfile1
-        +  RUN pip install --upgrade pip setuptools
-        -  RUN pip install --upgrade pip setuptools && \
-        -      pip install klio-exec
-
-        -  COPY job-requirements.txt job-requirements.txt
-        -  RUN pip install -r job-requirements.txt
+           RUN pip install --upgrade pip setuptools
+           RUN pip install -r job-requirements.txt
 
            COPY __init__.py \
+        -      setup.py \
+        -      MANIFEST.in \
                run.py \
                transforms.py \
                my-model.h5 \
                /usr/src/app/
 
-        -  ARG KLIO_CONFIG=klio-job.yaml
-        -  COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml
-
 .. collapsible:: Combined Example of Required & Suggested Changes
 
     .. code-block:: diff
 
-           FROM apache/beam_python3.6_sdk:2.23.0
+           FROM apache/beam_python3.6_sdk:2.24.0
 
            WORKDIR /usr/src/app
-        -  RUN mkdir -p /usr/src/config
 
            ENV GOOGLE_CLOUD_PROJECT my-project \
               PYTHONPATH /usr/src/app
 
-        -  RUN apt-get update && apt-get install -y libsndfile1
-        +  RUN pip install --upgrade pip setuptools
-        -  RUN pip install --upgrade pip setuptools && \
-        -      pip install klio-exec
-
-        -  COPY job-requirements.txt job-requirements.txt
-        -  RUN pip install -r job-requirements.txt
+          RUN pip install --upgrade pip setuptools
 
            COPY __init__.py \
-        +      setup.py \
-        +      MANIFEST.in \
-        +      job-requirements.txt \
-        +      my-model.h5 \
-        +      klio-job.yaml \
+        -      setup.py \
+        -      MANIFEST.in \
+               job-requirements.txt \
+               my-model.h5 \
+               klio-job.yaml \
                run.py \
                transforms.py \
                /usr/src/app/
 
-        -  ARG KLIO_CONFIG=klio-job.yaml
-        -  COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml
-        +  RUN pip install .
+           ARG KLIO_CONFIG=klio-job.yaml
+           COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml
+        -  RUN pip install .
+        +  RUN pip install -r job-requirements.txt
```

Comment:
And update these lines to reflect the same suggestion that I suggested above.

Also, make sure that the example code matches the prose, and that the "Combined Example of Required & Suggested Changes" actual matches the "Example of Required Changes" plus "Example of Suggested Changes."